lib/Common/Core/CmdParser.cpp

```diff
@@ -68,11 +68,12 @@ CmdLineArgsParser::ParseString(__inout_ecount(ceBuffer) LPWSTR buffer, size_t ce
             case ' ':
             case ',':
             case 0:
-                fDone = 1;
+                fDone = true;
                 break;
-            case '-':
+            case '-': // REVIEW is this correct? What syntax results in this path being taken?
```

Comment:
Could this be the case of parsing the ending `-` in disabling flag like `-OOPJIT-`?